src/download.rs

```diff
@@ -306,6 +306,36 @@ pub fn write_image(
     Ok(())
 }
 
+pub fn download_to_tempfile(url: &str, allow_insecure: bool, has_hash: bool) -> Result<File> {
+    if url.starts_with("http://") {
```

Comment:
reqwest doesn't insist on the trailing slashes.  Let's just match against `http:` or `https:`.

Reply:
That's... interesting. I'm not sure if we want to leak that through. I'd rather just be more strict here.